src/test/k6/requests/text.js

```diff
@@ -21,3 +21,53 @@ export function submitRandomTextAnswerExam(artemis, exercise, submissionId) {
     }
     return answer;
 }
+
+export function newTextExercise(artemis, exerciseGroup, courseID) {
+    const textExercise = {
+        maxPoints: 1,
+        title: 'text' + nextAlphanumeric(5),
+        type: 'text',
+        mode: 'INDIVIDUAL',
+    };
+
+    if (courseID) {
+        textExercise.course = { id: courseId };
+    }
+
+    if (exerciseGroup) {
+        textExercise.exerciseGroup = exerciseGroup;
+    }
+
+    const res = artemis.post(TEXT_EXERCISES, textExercise);
+    if (res[0].status !== 201) {
+        console.log('ERROR when creating a new text exercise. Response headers:');
+        for (let [key, value] of Object.entries(res[0].headers)) {
+            console.log(`${key}: ${value}`);
+        }
+        fail('FAILTEST: Could not create text exercise (status: ' + res[0].status + ')! response: ' + res[0].body);
+    }
+    console.log('SUCCESS: Generated new text exercise');
+
+    return JSON.parse(res[0].body);
+}
+
+export function assessTextSubmission(artemis, exerciseId, resultId) {
+    const assessment = [
+        {
+            credits: 4,
+            reference: 'Package:35f037f7-0606-4798-b43a-9bc76d741421',
+            referenceId: '35f037f7-0606-4798-b43a-9bc76d741421',
+            referenceType: 'Package',
+            text: 'AssessmentText',
+        },
```

Comment:
Should we use proper Text Feedback here?

Reply:
We are now using proper text feedback.